test/Chirp.Test/E2ETests.cs

```diff
@@ -80,4 +81,47 @@ public async Task DoesPrivateTimelineContainAdrianTest()
         content.Should().Contain("Adrian");
         content.Should().Contain("Hej, velkommen til kurset");
     }
+
+    [Fact]
+    public async Task RegisterNewUserTest()
+    {
+        // Arrange
+        var client = _factory.CreateClient();
+        
+        var getResponse = await client.GetAsync("/Identity/Account/Register");
+        var getContent = await getResponse.Content.ReadAsStringAsync();
+        
+
+        // Step 2: Parse the anti-forgery token from the page content
+        var tokenValue = ExtractAntiForgeryToken(getContent);
+        
+        var registerData = new Dictionary<string, string>
+        {
+            {"Input.Email","testuser@gmail.com"},
```

Comment:
Can you please change this so that we dont store passwords in plain text?

Reply:
To elaborate We dont want to store credentials in plain text this will be a vulnerability in the future

Reply:
The test that we've made doesn't actually store any values in the production DB, as it is an in memory test, thus the user created in this enviroment will be removed as it is stored in a dummy DB. 
Although you do have a point that it may be wise to store test variables that includes dummy users, if we want to actually create tests for production enviroment in the future using a specified user.

Reply:
As the other Nikolaj mentioned since it is a in memory test, it's data is scrapped after the test is done. And since this is a test dummy it's information like username and password does not leak any important information. But it is a good observation and we should of course keep this in mind when testing. So that we do use GitHub secrets for example, if we are handling sensitive data (for example a test which for some reason used an already exisisting user, then yes we should definitely not leak that user's data)